ddtrace/llmobs/_llmobs.py

```diff
@@ -564,15 +563,6 @@ def enable(
         config._llmobs_ml_app = ml_app or config._llmobs_ml_app
         config._llmobs_instrumented_proxy_urls = instrumented_proxy_urls or config._llmobs_instrumented_proxy_urls
 
-        # FIXME: workaround to prevent noisy logs when using the experiments feature
-        if config._dd_api_key and cls._app_key and os.environ.get("DD_TRACE_ENABLED", "").lower() not in ["true", "1"]:
-            log.debug(
-                "Tracing has been disabled: app key detected and DD_TRACE_ENABLED is not set to 'true' "
-                "(current value: DD_TRACE_ENABLED='%s')",
-                os.environ.get("DD_TRACE_ENABLED", ""),
-            )
```

Comment:
note - We'll look at fixing this long term, I'd prefer not to flood customer apps with noisy logs